src/main/java/org/reaktivity/nukleus/http2/internal/InternalSystemProperty.java

```diff
@@ -0,0 +1,48 @@
+package org.reaktivity.nukleus.http2.internal;
```

Comment:
The license plugin should have failed the build without the preamble at top of this source file.

Please review the `pom.xml` setting to make sure they are consistent with the other projects.